src/python/WMCore/Storage/StageInMgr.py

```diff
@@ -78,43 +73,53 @@ def initialiseSiteConf(self):
 
         """
 
-        implName = self.siteCfg.localStageOut.get("command", None)
-        if implName == None:
-            msg = "Unable to retrieve local stage out command\n"
-            msg += "From site config file.\n"
-            msg += "Unable to perform StageOut operation"
-            raise StageOutInitError( msg)
-        msg = "Local Stage Out Implementation to be used is:"
-        msg += "%s\n" % implName
-
-        pnn = self.siteCfg.localStageOut.get("phedex-node", None)
-        if pnn == None:
-            msg = "Unable to retrieve local stage out phedex-node\n"
-            msg += "From site config file.\n"
-            msg += "Unable to perform StageOut operation"
-            raise StageOutInitError( msg)
-        msg += "Local Stage Out PNN to be used is %s\n" % pnn
-        catalog = self.siteCfg.localStageOut.get("catalog", None)
-        if catalog == None:
-            msg = "Unable to retrieve local stage out catalog\n"
-            msg += "From site config file.\n"
-            msg += "Unable to perform StageOut operation"
-            raise StageOutInitError( msg)
-        msg += "Local Stage Out Catalog to be used is %s\n" % catalog
-
-        try:
-            self.tfc = self.siteCfg.trivialFileCatalog()
-            msg += "Trivial File Catalog has been loaded:\n"
-            msg += str(self.tfc)
-        except Exception as ex:
-            msg = "Unable to load Trivial File Catalog:\n"
-            msg += "Local stage out will not be attempted\n"
-            msg += str(ex)
-            raise StageOutInitError( msg )
+        self.stageOuts = self.siteCfg.stageOuts
+
+        msg = "\nThere are %s stage out definitions." % len(self.stageOuts)
```

Comment:
Same comment on this msg variable, which is only printed in case there is an exception. Can you please revisit it?

Reply:
Done